### PR TITLE
436 downloading attached files

### LINKED ISSUE #436: Implement endpoint for downloading attached files

### PR DESCRIPTION
**Pull request by alexkiro**
Closes #436

### CODE REVIEW

File: ozone/tests/factories.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
     HighAmbientTemperatureProduction,
     HighAmbientTemperatureImport,
     UploadToken,
-)
+    SubmissionFile)
```
Comment (sorinstelian) [inline ozone/tests/factories.py]: Really nitpicking, but let's keep a consistent style. :)

**Comment by sorinstelian** [comment]
Looks good; however tests seem to be failing.

File: ozone/tests/factories.py (hunk 1 above)
Comment (alexkiro) [reply ozone/tests/factories.py]: Fixed! :)

**Comment by alexkiro** [comment]
> Looks good; however tests seem to be failing.

Indeed, missed that those started failing after I added support for non-ascii files names. They ought to be passing now. I'll wait for Travis to finish his checks and will merge afterwards.

**Pull request merged**